src/sfctl/custom_cluster.py

```diff
@@ -39,6 +39,16 @@ def select_arg_verify(endpoint, cert, key, pem, ca, aad, no_verify):
     if pem and any([cert, key]):
         raise CLIError(usage)
 
+def show():
+    """
+    Display the current active endpoint.
+    """
+    from sfctl.config import (client_endpoint)
+    if client_endpoint() and client_endpoint().strip():
+        print("The current active endpoint is: " + client_endpoint())
```

Comment:
I think for the sake of globalisation and simplicity it's okay to return the endpoint, or empty string. Adding the text before will just make it more difficult for people to parse out the endpoint.

Reply:
good point.